packages/checkout/widgets-lib/src/components/Transactions/Transactions.tsx

```diff
@@ -301,6 +237,49 @@ export function Transactions({
     [],
   );
 
+  const handleWalletChange = useCallback(
+    async (event: WalletChangeEvent) => {
+      track({
+        userJourney: UserJourney.BRIDGE,
+        screen: 'EmptyStateNotConnected',
+        control: 'WalletProvider',
+        controlType: 'Select',
+        extras: {
+          walletProviderName: event.providerDetail.info.name,
+        },
+      });
+
+      try {
+        const web3Provider = new Web3Provider(event.provider as any);
+        const connectedProvider = await connectToProvider(checkout, web3Provider, true);
```

Comment:
should this changeAccount always hardcode to true here? I think this should just be for MetaMask right?

Reply:
Yeah, just tested, this is breaking when trying to use WalletConnect in the Transactions screen

Reply:
yep good catch fixed.